res/values-en/strings.xml

```diff
@@ -52,14 +52,14 @@
     <string name="pref_reset">Reset</string>
     <string name="pref_reset_summary">Use Default Settings to Replace the contents in /sdcard/rime</string>
 
-    <string name="pref_keyboard">Keyboard</string>
+    <string name="pref_preferences">Preferences</string>
```

Comment:
I use android 7.1.1, and I refer the AOSP not the Google input method.
I never install Google input method or use AOSP for privacy, especially I have trime.
The AOSP settting is similar with our sub preferences in Preferences, so I use this.
What about other choice or keep using keyboard???